src/collector/go.mod.ts

```diff
@@ -138,18 +139,20 @@ export class DependencyCollector implements IDependencyCollector {
 
     async collect(contents: string): Promise<Array<IDependency>> {
         let promiseExec = new Promise<Set<string>>((resolve, reject) => {
-            const vscodeRootpath = this.manifestFile.replace("file://", "").replace("/go.mod", "")
-            exec(getGoLangImportsCmd(),
-                { shell: process.env["SHELL"], windowsHide: true, cwd: vscodeRootpath, maxBuffer: 1024 * 1200 }, (error, stdout, stderr) => {
+            const sourceRootPath = url.fileURLToPath(this.manifestFile.replace("go.mod", ""));
```

Comment:
use dirname instead of ad-hoc replace. This shows the intent.

after 
```
import { dirname } from 'path';
```

```suggestion
            const sourceRootPath = url.fileURLToPath(dirname(this.manifestFile));
```